examples/diff.c

```diff
@@ -47,6 +47,15 @@ enum {
 	CACHE_NONE = 2
 };
 
+#define DEFAULT_THRESHOLD 50
+#define DEFAULT_BREAK_REWRITE_THRESHOLD 60
+#define USE_DEFAULT(X) ((X) == 0 || (X) > 100)
+
+struct pair {
+	int first;
+	int second;
+};
```

Comment:
Our style would be to pass these two as named output parameters from the function and use the return value as an `int` to signal errors.